src/Events/WishList/index.js

```diff
@@ -66,22 +72,57 @@ class WishList extends Component {
         });
     }
 
+    changeState = () => {
+        this.setState({ togglePastEvents: !this.state.togglePastEvents });
+    }
+
+    /**
+     * Count events data in array nested in two objects
+     * @param  {object} _events
+     * @return {number}
+     */
     countEventsByMonth = (_events) => {
         let monthEventsNumber;
-        _.forEach(_events, (group, key) => {
+        _.forEach(_events, (group) => {
             _.forEach(group, (e) => {
-                monthEventsNumber= +_.size(e);
+                monthEventsNumber = +_.size(e);
             });
         });
```

Comment:
You can try:
```
for(group of _events) {
    for(event of group) {
        ..
    }
}
```
Lodash has great helpers, but for-of can be very useful 🦃

Reply:
As discussed with @gergana95, for-of works great for arrays, but `_events` is an object. for-in is a little ugly in this case.
I'll try to make my next comment wiser.